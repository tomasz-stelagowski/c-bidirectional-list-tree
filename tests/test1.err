NODES: 2
NODES: 3
NODES: 4
NODES: 5
NODES: 6
NODES: 7
NODES: 7
NODES: 8
NODES: 8
NODES: 8
NODES: 7
NODES: 7
NODES: 7
NODES: 8
NODES: 8
NODES: 3
NODES: 4
NODES: 5
NODES: 6
NODES: 7
NODES: 7
NODES: 7
NODES: 3
NODES: 4
NODES: 5
NODES: 6
NODES: 7
NODES: 7
NODES: 7
NODES: 6
NODES: 7
NODES: 7
NODES: 7
NODES: 8
NODES: 8
NODES: 7
NODES: 7
NODES: 7
NODES: 7
NODES: 6
NODES: 6
NODES: 3
NODES: 3
NODES: 1
NODES: 1
