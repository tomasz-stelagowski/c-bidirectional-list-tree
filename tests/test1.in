ADD_NODE 0
ADD_NODE 0
ADD_NODE 2
ADD_NODE 2
ADD_NODE 0
ADD_NODE 2
RIGHTMOST_CHILD 0
ADD_NODE 1
RIGHTMOST_CHILD 2
RIGHTMOST_CHILD 4
DELETE_SUBTREE 5
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 7
ADD_NODE 6
RIGHTMOST_CHILD 1
DELETE_SUBTREE 2
ADD_NODE 0
ADD_NODE 0
ADD_NODE 7
ADD_NODE 1
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 1
DELETE_SUBTREE 1
ADD_NODE 10
ADD_NODE 0
ADD_NODE 9
ADD_NODE 10
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 10
DELETE_SUBTREE 16
ADD_NODE 10
RIGHTMOST_CHILD 10
RIGHTMOST_CHILD 15
ADD_NODE 0
RIGHTMOST_CHILD 0
DELETE_SUBTREE 18
RIGHTMOST_CHILD 13
RIGHTMOST_CHILD 9
RIGHTMOST_CHILD 0
DELETE_SUBTREE 14
RIGHTMOST_CHILD 0
DELETE_SUBTREE 10
RIGHTMOST_CHILD 0
DELETE_SUBTREE 9
RIGHTMOST_CHILD 0
