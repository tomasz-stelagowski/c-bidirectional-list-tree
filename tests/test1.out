OK
OK
OK
OK
OK
OK
5
OK
6
-1
OK
2
-1
OK
7
OK
OK
OK
OK
OK
10
12
OK
OK
OK
OK
OK
14
16
OK
OK
17
-1
OK
18
OK
-1
15
14
OK
10
OK
9
OK
-1
